app/src/main/java/ar/com/wolox/android/example/model/LoginResponse.kt

```diff
@@ -0,0 +1,19 @@
+package ar.com.wolox.android.example.model
+
+data class LoginResponse(
+    val data: Data,
+    val success: Boolean,
+    val errors: List<String>
+)
+
+data class Data(
```

Comment:
Perhaps the name `Data` it's not very explanatory. Also, when you are declaring a data class to handle the json object from the request, you can use `@SerializedName("varName")` to reference certain field from the object, and you can use your own variable name. 
```
data class TransactionModel(
        @SerializedName("idReferencia") val id: String = "",
        @SerializedName("fecha") val date: String = "",
        @SerializedName("detalle") val detail: String = "",
        @SerializedName("importe") val amount: String = "",
)
```